server/views/components/applicationSummary/macro.njk

```diff
@@ -1,48 +1,55 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "../../components/printButton/macro.njk" import printButton %}
+
 {% macro applicationSummary(summary) %}
-  <div id="application-summary">
-    <h2 class="govuk-heading-m">
-      Application summary
-    </h2>
+  <div id="application-summary" class="submitted-application-panel govuk-panel govuk-!-margin-bottom-7">
 
-    <ul class="govuk-list govuk-list--spaced">
-        <li>
-          <span class="govuk-!-font-weight-bold">Application ID: </span>{{ summary.id }}
-        </li>
-      {% if summary.name %}
-        <li>
-          <span class="govuk-!-font-weight-bold">Applicant name: </span>{{ summary.name }}
-        </li>
-      {% endif %}
-      {% if summary.prisonNumber %}
+    <h1 class="govuk-panel__title govuk-!-margin-bottom-5">
```

Comment:
sounds like we need to add back in the Application ID https://mojdt.slack.com/archives/C04T6SC31HA/p1707239412686999?thread_ts=1707239368.336179&cid=C04T6SC31HA

Reply:
great spot, I have done now